aidants_connect_web/views/service.py

```diff
@@ -53,22 +55,56 @@ def habilitation(request):
     return render(request, "public_website/habilitation.html")
 
 
+def mandats_chart(request):
+    demarches_count = []
+
+    autorisation_use = Journal.objects.excluding_staff().filter(
+        action="use_autorisation"
+    )
+
+    for demarche in settings.DEMARCHES.keys():
+        demarches_count.append(
+            {
+                "title": demarche,
+                "icon": settings.DEMARCHES[demarche]["icon"],
+                "value": autorisation_use.filter(demarche=demarche).count(),
+            }
+        )
+
+    demarches_count.sort(key=lambda x: x["value"], reverse=True)
```

Comment:
Pour trier sur une clé, j'aime bien utiliser `operator.attrgetter` plutôt que me trimballer une `lambda` : 

```suggestion
    from operator import attrgetter
    demarches_count.sort(key=attrgetter("value"), reverse=True)
```

Reply:
Ah comme ce sont des `dict` en fait il faut utiliser itemgetter : 

```python
    from operator import itemgetter
    demarches_count.sort(key=attrgetter("value"), reverse=True)
```